src/libre/parser.act

```diff
@@ -31,6 +30,8 @@
 	#include <ctype.h>
 
 	#include "libfsm/internal.h" /* XXX */
+	#include "libre/re_ast.h"
+	#include "libre/re_char_class.h"
 
 	#include <fsm/fsm.h>
 	#include <fsm/bool.h>
```

Comment:
Can `<fsm/*.h>` and `"libfsm/internal.h"` go now?

Reply:
Yes they can! Removed in 07127fa.